src/sidebar/components/slider.js

```diff
@@ -2,6 +2,12 @@ import { createElement } from 'preact';
 import { useCallback, useEffect, useRef, useState } from 'preact/hooks';
 import propTypes from 'prop-types';
 
+/**
+ * @typedef SliderProps
+ * @prop {Object} [children] - The slidable content in question.
```

Comment:
```suggestion
 * @prop {Object} [children] - The slideable content in question.
```

As a suggestion, maybe "The slideable content to hide or reveal" might be clearer?

Reply:
:)

https://wikidiff.com/slideable/slidable#:~:text=As%20adjectives%20the%20difference%20between,slid%2C%20or%20made%20to%20slide.